sys/random/mersenne.c

```diff
@@ -129,7 +133,12 @@ uint32_t genrand_uint32(void)
     y ^= (y << 7) & 0x9d2c5680UL;
     y ^= (y << 15) & 0xefc60000UL;
     y ^= y >> 18;
+
+#ifdef MODULE_ENTROPY
+    return y + entropy;
```

Comment:
Hi! I am no mathematician...
Instead of `return y + entropy;` (which can lead to overflows; and from a distribution/security point of view I am not sure if an adittion adds entropy to a source, I would say no.. and on the contrary you will likely have less number having a value lower than MAXINT/4),
why don't you perform an xor? `return y ^ entropy;` 
I think it nicely uses the `entropy` (as a one time pad) to modify `y`.